build.sbt

```diff
@@ -29,8 +29,8 @@ lazy val scallion = project
     target in Compile in doc := baseDirectory.value / "docs",
 
     libraryDependencies ++= Seq(
-      "org.scalatest" %% "scalatest" % "3.0.8" % "test",
-      "ch.epfl.lara" %% "silex" % "0.5" % "test",
+      "org.scalatest" %% "scalatest" % "3.2.9" % "test",
+      "ch.epfl.lara" %% "silex" % "0.6" % "test",
```

Comment:
Silex was made available on Bintray previously. Is there a convenient way to distribute it now?

Reply:
For now, we have to work with `publishLocal`...

Reply:
@redelmann Do you think the process for publishing it is different with Scala 3? Can't the new version of Silex be published as before? It's quite important that students not have to do `publishLocal` before compiling their projects.

Reply:
Bintray simply shut down. Better than publish local, simply put silex and scallion jars in a lib folder?

Reply:
Oh I see! Yes, some jars in a lib folder should do fine. Would be nice to publish on Sonatype/Maven Central, but that's a pain to set up.

Reply:
The current Scallion version I downloaded here gives the silex not found error when compiling.
Downloaded Silex as well and then publishLocal solved the issue.

Reply:
Yes, we need to include silex for the _examples_ project to work. I would suggest we add a lib folder for that specific project with the silex jar.